llvm/lib/Target/X86/X86InstrSystem.td

```diff
@@ -520,6 +520,7 @@ let SchedRW = [WriteSystem] in {
     } // Defs SSP
   } // Uses SSP
 
+let Predicates = [NoEGPR] in {
```

Comment:
How about other CET instuctions like `RDSSPD`? It has GPR operand too. It looks to me strange the symmetry instuction doesn't gain APX promotion.

Reply:
`RDSSPD` is not promoted. I guess it's probably b/c it's used by intrinsic and the dest reg is `RAX` in most cases.